_includes/navigation.html

```diff
@@ -1,6 +1,7 @@
 <a href="./volunteer.html">Volunteer</a>
 <a href="./about.html">About Us</a>
 <a href="./projects.html">Projects</a>
+<a href="./codeofconduct.html">Code Of Conduct</a>
```

Comment:
If I’m not mistaken, this will appear in both the header and the footer. The additional link will cause the row of links in the header to get pretty cramped:

<img width="2160" height="416" alt="Header with Code of Conduct after Projects, no spacing between any link." src="https://github.com/user-attachments/assets/1a36cc23-3186-4b5c-a4d6-4e9d59123d90" />

Would it be OK to only link to the code of conduct in the footer? Add the link right below this include directive:

https://github.com/codeforsanjose/codeforsanjose.github.io/blob/f1f56f4c9eb059fd93b1fdad6c3915ace8a0f7e7/_includes/footer.html#L4

(While you’re at it, the CoC capitalizes it as “Code of Conduct”, so we should be consistent with that spelling.)

Reply:
got it